src/Be.Vlaanderen.Basisregisters.Projector/ConnectedProjections/IConnectedProjectionsManager.cs

```diff
@@ -49,5 +50,12 @@ public interface IConnectedProjectionsManager
         /// </summary>
         /// <param name="name">Case insensitive name of the projection to check.</param>
         bool Exists(string name);
+
+        /// <summary>
+        /// Get the projection state.
```

Comment:
either this should be `states` or the method should be `GetProjectionState`, I'm guessing it's the documentation which needs to change